install.sh

```diff
@@ -177,6 +177,23 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_bash_completion() {
+  if [ -f "$HOME/.bashrc" ] || [ -f "$HOME/.bash_profile" ] || [ -f "$HOME/.zshrc" ] ; then
```

Comment:
I wonder if it would be more efficient to do everything in the `for` loop below, including downloading the `bash_completion` file on the first iteration?